run.py

```diff
@@ -1,24 +1,23 @@
 import logging
 import logging.config
-logging.config.fileConfig('logging.conf')
-import os
-import json
 import datetime
-
+import requests
+from json import dumps
+from os import getenv
 from os.path import join
 
 from hdx.api.configuration import Configuration
 from hdx.data.dataset import Dataset
 from hdx.utilities.downloader import Download
 from hdx.utilities.path import temp_dir
+from hdx_redis_lib import connect_to_hdx_event_bus_with_env_vars
 
 from check_location import check_location, get_global_pcodes
-
-from hdx_redis_lib import connect_to_hdx_event_bus_with_env_vars
 from helper.facade import facade
 from helper.ckan import patch_resource_with_pcode_value
 from helper.util import do_nothing_for_ever
 
+logging.config.fileConfig("logging.conf")
```

Comment:
I know it looks ugly, but this line needs to be as high as possible because it configures the logging and other imports (expecially from the hdx python lib) are generating logs.